api/src/main/java/org/pmiops/workbench/institution/InstitutionService.java

```diff
@@ -132,4 +132,7 @@ boolean validateInstitutionalEmail(
    * @return
    */
   List<UserTierEligibility> getUserTierEligibilities(DbUser user);
+
+  /** Should the user be allowed to bypass the expiration of their initial credits? */
```

Comment:
this reads to me like it's an action the user can choose to take.  Instead, the user is not exposed to the concept of expiration, because it doesn't apply to them.

Reply:
Good point. I updated the wording:
"If true, this institution's users are exempt from credit expiration."

How does that sound?